packages/auth-construct/API.md

```diff
@@ -48,7 +47,7 @@ export type AuthProps = {
         externalProviders?: ExternalProviderOptions;
     };
     senders?: {
-        email: Pick<UserPoolSESOptions, 'fromEmail' | 'fromName' | 'replyTo'> | CustomEmailSender;
+        email: Pick<UserPoolSESOptions, 'fromEmail' | 'fromName' | 'replyTo'>;

```

Comment:
The branch's parent is the current HEAD of main.
```
> git rev-parse HEAD~2
06e4def3eded2e4127c4501a4dce669abba900db
```

I'm assuming the API changes were necessary because running `npm i` to update the package-lock after bumping the `graphql-generator` version pulled in newer versions of other packages that aren't in main.

What do you suggest?

Reply:
That's not possible.
The changes in `API.md` files are related to code in the repo that was added recently.

`npm run clean && npm install && npm run build && npm run update:api` should help. (gonna push the change since I have it locally).

`API.md` files are generated from build artifacts. If local workspace is stale then it may give unexpected deltas.

Reply:
> If local workspace is stale then it may give unexpected deltas.

that is the reason we have check in PR build. we haven't found better way to force consistency.